tml/constants.py

```diff
@@ -75,3 +75,4 @@
 }
 
 SPEEDUPINDEX = 28
+R_TILEINDEX = dict([(v, k) for k, v in TILEINDEX.items()])
```

Comment:
Because the list is not exhaustive maybe we should just always get the image.

Reply:
Which image are you talking about ?